crates/biome_formatter/src/lib.rs

```diff
@@ -176,6 +179,20 @@ impl Default for LineWidth {
     }
 }
 
+impl Deserializable for LineWidth {
+    fn deserialize<L: biome_deserialize::DeserializableLanguage>(
+        value: SyntaxNode<L>,
+        diagnostics: &mut Vec<DeserializationDiagnostic>,
+    ) -> Option<Self> {
+        L::deserialize(
+            value,
+            UnsignedIntegerVisitor::new(Self::MIN, Self::MAX),
```

Comment:
Because `LineWidth` has a custom maximum (320). This ensures that the value is not greater than 320. SHould we remove this limit? I am not sure why we use 320. That's pretty low?

Reply:
I can't remember unfortunately